src/SshKeyScan.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Spatie\Ssh;
+
+use InvalidArgumentException;
+use Symfony\Component\Process\Exception\ProcessFailedException;
+use Symfony\Component\Process\Process;
+
+class SshKeyScan
+{
+    private string $sshPort = '22';
+
+    private string $keyType = 'rsa';
+    /**
+     * @var string
+     */
+    private $path;
```

Comment:
Use PHP 7.4 style typing, do not use docblocks. If you need an instance variable that can't be set in the constructor or initialised properly there, make it optional.